tailor_client/src/dbus/profiles.rs

```diff
@@ -20,6 +20,12 @@ trait Profiles {
 
     async fn get_active_profile_name(&self) -> fdo::Result<String>;
 
+    async fn get_active_performance_profile_name(&self) -> fdo::Result<String>;
+
+    async fn set_active_performance_profile_name(&self, name: &str) -> fdo::Result<()>;
+
```

Comment:
So far, changing a LED or fan profile was done through `add_profile` -> `set_active_profile_name` -> `reload`. Setting it manually does not make sense in this context because then the performance profile would be independent from the profile stored in the config files.

Also, performance profiles should probably have their own DBUS interface, such as fans and LEDs: https://github.com/AaronErhardt/tuxedo-rs/blob/main/tailord/src/dbus/fan.rs
The "Profiles" interface was only used for managing configurations and loading them so far.

Reply:
Ok got it. I'll change this.